modules/acid/src/main/ts/ephox/acid/gui/components/SaturationBrightnessPalette.ts

```diff
@@ -19,7 +20,11 @@ export interface SaturationBrightnessPaletteSketcher extends Sketcher.SingleSket
   setThumb: (slider: AlloyComponent, hex: Hex) => void;
 }
 
-const paletteFactory = (_translate: (key: string) => string, getClass: (key: string) => string): SaturationBrightnessPaletteSketcher => {
+const paletteFactory = (translate: (key: string) => string, getClass: (key: string) => string): SaturationBrightnessPaletteSketcher => {
+  const translateReplace = (key: string, value1: number, value2: number): string => {
+    return translate(key).replace('{0}', value1 + '').replace('{1}', value2 + '');
+  };
```

Comment:
You don't need to replace manually, translate actually takes an array - although the `ColorPicker` translate function will need to be updated to leverage this, that's where the `key: string` type is enforced.
https://github.com/tinymce/tinymce/blob/ece3bf774b9ae20910d073af5351f162c4fb52d5/modules/tinymce/src/core/main/ts/api/util/I18n.ts#L122-L126

Here's how it's used:
https://github.com/tinymce/tinymce/blob/46ae40b9e7dbab275fd0731ca34ed0ea0633e475/modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts#L30